src/core/pageModules/StepsModule/StepsModule.tsx

```diff
@@ -29,6 +29,19 @@ export function StepsModule({
   type,
   className,
 }: StepsModuleProps) {
+  const stepsContents = steps?.map((step, index) => {
+    // The card module does not contain any proeprty that could be used as an unique id, so one needs to be created
+    const uniqueKey = `card-${index}-${step.title?.substring(
+      0,
+      4,
+    )}-${step.content?.substring(0, 4)}`;
```

Comment:
How about making this more robust by using a hash of the tuple of the input fields that should together suffice to make the key unique—hashing only to make the result smaller, the uniqueness property should come from the tuple of the selected input fields (e.g. in [sports](https://github.com/City-of-Helsinki/events-helsinki-monorepo/blob/sports-helsinki-v1.12.0/apps/sports-helsinki/src/domain/venue/venueHighlights/VenueHighlights.tsx#L18-L23))?

For example:
```typescript
import { createHash } from 'crypto';
...
const uniqueKey = createHash('sha256').update(`${step.title}: ${step.content}`).toString();
```